packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -275,3 +262,88 @@ describe('listAccountHistory', () => {
     expect(history.find((h) => h.type === 'AccountToUtxos' && h.amounts[0] === '-3.97000000@DFI')).toBeTruthy()
   })
 })
+
+describe('listAccountHistory for poolpair', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  const createToken = createTokenForContainer(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await container.waitForWalletBalanceGTE(200)
+    const address = await container.call('getnewaddress')
+    await container.call('utxostoaccount', [{ [address]: '100@0' }])
+    await createToken(address, 'DDAI', 2000)
+    await createPoolPair('DDAI')
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  async function createPoolPair (tokenB: string, metadata?: any): Promise<void> {
+    const poolAddress = await container.call('getnewaddress')
+    const defaultMetadata = {
+      tokenA: 'DFI',
+      tokenB,
+      commission: 0,
+      status: true,
+      ownerAddress: poolAddress
```

Comment:
ownerAddress: await container.call('getnewaddress')

Reply:
Made changes to this